packages/react-components/react-migration-v0-v9/src/stories/Spinner/Description.md

```diff
@@ -0,0 +1 @@
+The Spinner Mixin adds styles to match V0 Loader.
```

Comment:
```suggestion
The Spinner Mixin adds styles to match `@fluentui/react-northstar` [Loader](https://fluentsite.z22.web.core.windows.net/0.65.0/components/loader/definition).
```

we only refer to v0 internally and maybe it would be nice to add a link to the v0 docs